pset-iterators.js

```diff
@@ -66,6 +72,12 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+ const presidentsNamedJames = presidents.filter((president) => {
+   return president.president.split(' ')[0] === 'James'
```

Comment:
@kcycle nice work here, using `.split()` to isolate the first name and match against that